src/configure/helper/LocalGitRepoHelper.ts

```diff
@@ -202,4 +198,13 @@ export class LocalGitRepoHelper {
         const buf = fs.readFileSync(pathToFile);
         return buf.toString();
     }
+
+    public async addContentToFile2(content: string, pathToFile: string): Promise<string> {
+        fs.writeFileSync(pathToFile, content);
```

Comment:
this is a single statement and can be made inline. Does not require a separate function.
If at all you want a separate function, then give a better name.

Reply:
Actually ``LocalGitRepoHelper`` class we use to handle things related to files and dir operations. So i have made this function to keeps consistency. I have rename this function to ``writeFileContent``